### ISSUE TITLE
ToggleButton: WCAG standards

### ISSUE THREAD

**Issue by matuzo** [primefaces/primereact#1919]
In [your docs](https://www.primefaces.org/primereact/showcase/#/accessibility) it says:

> UI Components for everyone with **full support** for Web Content Accessibility Guidelines (WCAG).

As already[ pointed out by another user](https://github.com/primefaces/primereact/issues/1887), this isn’t true. Neither your website nor _all_ components are WCAG 2.0 or 2.1 compliant. Since some companies base their choice for a specific library on the fact whether or not it’s WCAG compliant, it would be important to either rephrase the statements on the page and list all inaccessible components or to provide us with the information which expert checked which version of the library against which version of WCAG.

Thank you

**Comment by marcus-herrmann** [comment]
An example: Your toggle button component is literally a `<div>` with `tabindex="0"`, which is a violation of WCAG 4.1.2 at a minimum.

[Heydon Pickering wrote about the proper implementation of an accessible ToggleButton four years ago](https://inclusive-components.design/toggle-button/).

**Comment by melloware** [comment]
Because this ticket was generic I made it specifically for ToggleButton as other WCAG standards are being tackled for 8.1.0

**Comment by habubey** [comment]
Issue tracker is used for defects only as part of our commitment to quality and continuous improvement in all areas. Enhancements are collected as valuable community feedback and managed internally so moving this enhancement ticket to our internal project management backlog.

**Comment by marcus-herrmann** [comment]
When your marketing material promises WCAG compliance, but the components do not deliver, I myself see this as a defect and not some "enhancement".

**Comment by habubey** [comment]
Accessibility features will start to be added this quarter. Expected to be ready and finished by the end of the year
